ReadME.md

```diff
@@ -1,5 +1,13 @@
 1. cd into the 'APPRENTICETAKEHOME' folder
+<<<<<<< HEAD
```

Comment:
Heads up you have some merge conflicts here.

Reply:
@leonpham0 Thanks for the heads up! I fixed it. :)